users/users/login.php

```diff
@@ -53,6 +53,12 @@ public function keygen()
 			$id = $model->getUserId('email', $username);            
 		}
 
+		$res					=	array();
```

Comment:
@UmeshBhosale  In the starting of the file, following list of imports is written.

jimport('joomla.plugin.plugin');
jimport('joomla.html.html');
jimport('joomla.application.component.controller');
jimport('joomla.application.component.model');
jimport('joomla.user.helper');
jimport('joomla.user.user');
jimport('joomla.application.component.helper');

I dont thinnk all of these are needed